google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/JsonToProtoMessage.java

```diff
@@ -235,6 +239,16 @@ private static void fillRepeatedField(
         case BYTES:
           if (val instanceof String) {
             protoMsg.addRepeatedField(fieldDescriptor, ((String) val).getBytes());
+          } else if (val instanceof JSONArray) {
+            try {
+              byte[] bytes = new byte[((JSONArray) val).length()];
+              for (int j = 0; j < ((JSONArray) val).length(); j++) {
+                bytes[j] = (byte) ((byte) (((JSONArray) val).get(j)) & 0xFF);
```

Comment:
We should add a unit test for this.

Reply:
I've added a test, adding it in to the existing test proved very complicated since it required a repeated set of BYTE insertions. I just made a similar test that stands alone to cover these lines.